airflow/providers/google/cloud/operators/cloud_build.py

```diff
@@ -15,121 +15,980 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Operators that integrate with Google Cloud Build service."""
+
+"""Operators that integrates with Google Cloud Build service."""
+
 import json
 import re
+import warnings
 from copy import deepcopy
-from typing import Any, Dict, Optional, Sequence, Union
+from typing import Any, Dict, Optional, Sequence, Tuple, Union
 from urllib.parse import unquote, urlparse
 
-try:
-    import airflow.utils.yaml as yaml
-except ImportError:
-    import yaml
+import yaml
+from google.api_core.retry import Retry
+from google.cloud.devtools.cloudbuild_v1.types import Build, BuildTrigger, RepoSource
```

Comment:
I am working now locally on fixing system tests and this change broke a system test `CloudSqlProxySystemTest::test_run_example_dag_cloudsql_query` from file `test_cloud_sql_system.py` for me.
I was able to run it with 'Passed' status before  but now the error I am getting here is:
```
ERROR    airflow.models.dagbag.DagBag:dagbag.py:329 Failed to import: /opt/airflow/airflow/providers/google/cloud/example_dags/example_cloud_build.py
Traceback (most recent call last):
  File "/opt/airflow/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/airflow/providers/google/cloud/example_dags/example_cloud_build.py", line 38, in <module>
    from airflow.providers.google.cloud.operators.cloud_build import (
  File "/opt/airflow/airflow/providers/google/cloud/operators/cloud_build.py", line 30, in <module>
    from google.cloud.devtools.cloudbuild_v1.types import Build, BuildTrigger, RepoSource
ModuleNotFoundError: No module named 'google.cloud.devtools'
```
I also cannot find this package publicly available in pypi.
Is this some internal package not available for others? Am I doing something wrong?

Reply:
Hey @mnojek , google.cloud.devtools.cloudbuild_v1 is from here:
https://github.com/apache/airflow/blob/main/setup.py#L296

Could you try rebuilding your docker image or do `pip install google-cloud-build`?

Reply:
Installing this package helped. Thanks!